.github/workflows/integration-local.yml

```diff
@@ -29,28 +32,15 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Root NPM Install
-        run: npm i && npx lerna exec npm install
+        run: npm i fs-jetpack chalk && npm i && npx lerna exec npm install
```

Comment:
Might need a comment about why it is done (circular dependency with the preinstall script in package.json)